lib/plug/parsers/json.ex

```diff
@@ -43,6 +43,17 @@ defmodule Plug.Parsers.JSON do
     {:ok, %{}, conn}
   end
 
+  defp decode({:ok, body, conn}, {module_name, function_name, extra_args}) do
+    case apply(module_name, function_name, [body, extra_args]) do
```

Comment:
I wonder if this should be `[body, extra_args]` or `[body | extra_args]`?

Reply:
Yup!

Reply:
Definitely `[body | extra_args]`!  ^.^

Reply:
Good catch! 👍